test-case/verify-kernel-boot-log.sh

```diff
@@ -23,4 +23,25 @@ disable_kernel_check_point
 
 print_module_params
 
+# Check this device time is NTP Synchronized
+if checkNTPSync; then
+    printf '\nTime Check: NTP Synchronized\n'
+else
+    timedatectl show
+    # try to disable/enable NTP once, this will trigger time sync twice,
+    # before stopping npt and after enabling ntp.
+    # this might fix the problem
+    reenableNTPSync
+
+    # wait for 5 seconds
+    sleep 5
```

Comment:
Is this based on personal experience or some documentation? Both? Either way it's important to document every timeout.

Reply:
Did you offset the clock on purpose and test this?

Reply:
Yes I tested this after offset the clock by a few hours. It worked fine.
For sleep I put some sleep to make sure that system time is synchronized but actually this is not necessary. Will remove it.

Reply:
Thanks, please explain what you tested in the comments, err on the verbose side. These issues are very painful and time-consuming to test so sharing experiences is definitely worth it.